garrysmod/gamemodes/terrortown/gamemode/traitor_state.lua

```diff
@@ -180,5 +179,7 @@ local function force_spectate(ply, cmd, arg)
    end
 end
 concommand.Add("ttt_spectate", force_spectate)
-
+net.Receive("TTT_Spectate", function(l, pl)
+   force_spectate(pl, nil, {net.ReadInt(2)})

```

Comment:
If you decide to go with net.WriteBit for the net send, then you will need to change this line too.

**Change:**
force_spectate(pl, nil, {net.ReadInt(2)})

**To:**
force_spectate(pl, nil, {net.ReadBit()})